subsquid/src/model/generated/pabloPool.model.ts

```diff
@@ -54,7 +54,7 @@ export class PabloPool {
   @Column_("numeric", {transformer: marshal.bigintTransformer, nullable: false})
   calculatedTimestamp!: bigint
 
-  @OneToMany_(() => PabloPoolAsset, e => e.pool, {eager: true})
```

Comment:
looks like an unintended change, let's revert. Also let's run the unit tests locally for the subsquid stuff as they are not yet running automatically on CI

Reply:
This is automatically generated by Subsquid using TypeORM, so it's not something we should hard-code. As nothing changed on the schema regarding this type, it might be something that they changed internally

Reply:
I know it's automatically generated, but I remember having to change to get data loaded for some logic for pablo. Better run all tests and verify.